content/blog/2018/12/2018-12-10-outreachy-audit-log-plugin.adoc

```diff
@@ -0,0 +1,23 @@
+---
+:layout: post
+:title: Outreachy internships to add audit logging support to Jenkins
+:tags:
+- community
+- outreachy
+- outreachy2018
+:author: jvz
+---
+This year marks the first time the Jenkins Foundation is participating in https://www.outreachy.org/[Outreachy].
+Outreachy is a program vaguely similar to Google Summer of Code (GSoC) where interns work on open source projects for a paid stipend.
+The key difference is that Outreachy reaches out to underrepresented minorities and those who face systemic bias or discrimination in the technology industry in their home country.
+Once I learned about this program, I immediately volunteered to mentor as the concept strongly aligns with my ideals of inclusiveness and community building.
+I'm happy to report that both the Jenkins Foundation and CloudBees have been very supportive of this program.
+
+Expanding on our previous efforts to mentor students in GSoC, this year we've joined up with Outreachy to mentor two interns this year.
+Joining with me to implement https://github.com/jenkinsci/audit-log-plugin[audit logging support for Jenkins] are https://github.com/Lathaguna[Latha Gunasekar] and https://github.com/davidolorundare[David Olorundare].
+These two will be learning a lot about both professional software engineering as well as open source community engagement.
+Stay tuned for blog post entries introducing both people in the near future.
+
+This project forms a new connection between Jenkins and https://logging.apache.org/log4j/2.x/[Apache Log4j] which offers great opportunities for our interns to learn more about open source governance and meet new people.
+As a bonus, this audit logging project aims to provide the tooling necessary to support advanced obervability concerns such as running anamoly detection on authentication events to detect potential intrusion attempts.
+We will also be authoring a https://github.com/jenkinsci/jep[JEP] to detail the audit logging API provided by the plugin and how other plugins can define and log their own audit events besides the Jenkins Core ones that come with the plugin.
```

Comment:
I suggest a brief wrap up statement in its own paragraph "I'm looking forward to the great work we'll be doing together, and hope we're able to welcome more Outreachy interns soon!"